src/components/NftRewards/NftRewardsSection.tsx

```diff
@@ -127,8 +128,11 @@ export function NftRewardsSection() {
         <RewardTiersLoadingSkeleton />
       ) : (
         <div
-          // hax to make scrollbars look nice
-          className="-mt-3 -ml-3 -mr-5 max-h-[500px] overflow-auto pb-3 pt-3 pl-3 pr-5"
+          className={twJoin(
+            // hax to make scrollbars look nice
+            '-mt-3 -ml-3 -mr-5 overflow-auto pb-3 pt-3 pl-3 pr-5',
+            isMobile ? 'max-h-[950px]' : 'max-h-[620px]',
```

Comment:
no need to use `isMobile`. Can use media query instead:

```suggestion
            'max-h-[950px] md:max-h-[620px]',
```